src/pages/Task.tsx

```diff
@@ -108,6 +99,11 @@ const TaskCore: React.FC = () => {
       dispatchBanner.errorBanner(
         `There was an error loading the task: ${err.message}`
       ),
+    onCompleted: () => {
+      setFailedTask(
+        task?.status?.includes("failed") || task?.status?.includes("timed-out")
```

Comment:
this can be clearer if we directly compare the task status against [task status constants defined in spruce](https://github.com/evergreen-ci/spruce/blob/master/src/types/task.ts#L49)

Reply:
I also don't think this will work as expected since onCompleted is creating a closure around the `task` variable from line 109. You would want to look at the data object passed in a the first argument to `onCompleted`. https://www.apollographql.com/docs/react/data/queries/#options